src/components/Checkout/OrderTotals.vue

```diff
@@ -79,28 +79,14 @@
 
 		<div v-if="showPromoCreditTotal">
 			<div
-				class="tw-text-h3 tw-mb-2 tw-text-left md:tw-text-right
+				class="tw-mb-2 tw-text-left md:tw-text-right
 			tw-flex tw-justify-end tw-items-center" data-testid="basket-promo-total"
 			>
-				<span class="tw-w-full">
```

Comment:
Folks can add a Redemption Code or Have lending rewards or even UPC codes outside of managed lending campaigns and they are not required to use that credit for their current loan purchase. This means there can still be an available amount that isn't applied.

Thus, we'll need to keep some display of the available credit if that computed property has a value.

Reply:
Okay got it -- next thing I plan to work on is incorporating a flag that enables the order totals component to know if it's on a campaign page so we can retain the way these totals are displayed but swap it out for the new design on the cc pages.  But for now I've reinstated the old order totals so that kiva.org checkout won't be affected.

Reply:
okay, I've since added that flag now.